ansible/roles/fuseki/tasks/main.yml

```diff
@@ -0,0 +1,93 @@
+
+- name: Download Fuseki tarball
+  get_url:
+    url: https://archive.apache.org/dist/jena/binaries/apache-jena-fuseki-2.3.0.tar.gz
```

Comment:
The version number is hardcoded in a number of places. It would be better to define it in only one place.
Version 2.3.0 is pretty old, 3.8.0 is the latest version of Fuseki1 (and final, since new development is done only on Fuseki2).